api/client/cluster/client.go

```diff
@@ -318,6 +318,11 @@ func (c *clusterClient) GetGossipState() *cluster.ClusterState {
 	return status
 }
 
+func (c *clusterClient) GetGossipIntervals() types.GossipIntervals {
+	// TODO: do we need to implement this?
```

Comment:
No we don't need to implement it since there is no need to expose this info over the REST client API.